samza-test/src/test/java/org/apache/samza/test/operator/TestRepartitionJoinWindowApp.java

```diff
@@ -38,7 +38,7 @@
 import org.junit.Assert;
 import org.junit.Test;
 
-import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.*;
```

Comment:
Please do not use *

Reply:
that's not me :)  Intelij reformat. Fixed.